lib/config.ts

```diff
@@ -274,9 +283,20 @@ export function get(name: string): any
                 convictConf.get('https.cert'))),
             ca: fs.readFileSync(path.resolve(__dirname,
                 convictConf.get('https.ca'))),
+            crl: fs.readFileSync(path.resolve(__dirname,
+                convictConf.get('https.crl'))),
             requestCert: true,
             rejectUnauthorized: true
         };
+
+        // Setup file watch for crl changes
+        fs.watch(path.resolve(__dirname, convictConf.get('https.crl')),
+                 (event: string, filename: string): void => {
+                    // Re-read the crl file
+                    otherConf['serverOptions'].httpsServerOptions.crl =
+                        fs.readFileSync(path.resolve(__dirname, convictConf.get('https.crl')));
+                    emitter.emit('change', 'https.crl');  // Signal the server that config changes
+                });
```

Comment:
According to [Node document:](http://nodejs.org/api/fs.html#fs_fs_watch_filename_options_listener)

```
Filename Argument#

Providing filename argument in the callback is not supported on every platform (currently it's only supported on Linux and Windows). Even on supported platforms filename is not always guaranteed to be provided. Therefore, don't assume that filename argument is always provided in the callback, and have some fallback logic if it is null.
```
But I guess I can at least cache the value in a variable so we don't need to read it multiple time.